Testscripts/Linux/SRIOV-ModuleReload.sh

```diff
@@ -60,6 +66,12 @@ mlnx_sts=$(echo "$module_name_in_use" | grep -c mlx)
 if [ "$mlnx_sts" -eq 1 ]; then
     module_version=$(echo "$module_name_in_use" | sed  's/_/ /g' | awk '{print $1}')
     primary_module="${module_version}_en"
+    target_mod_check=$(lsmod | grep -i $primary_module)
+    LogMsg "Checking $primary_module in lsmod and ensure if it is primary - $target_mod_check"
+    # Sometimes mlx OFED driver has mlx5_ib not mlx5_en
+    if [ $? -eq 1 ]; then
```

Comment:
This $? will be the return value for LogMsg, always 0?

Reply:
Fixed in 1f3f9519818c765dcdd3b3281ae3a6b66d1e1e04